src/Tribe/Editor/Objects/Event.php

```diff
@@ -59,6 +60,20 @@ public function data( $key = null, $default = null ) {
 				'is_new_post' => true,
 			];
 
+			$start_date = tribe_get_request_var( 'tribe-start-date' );
+
+			/**
+			 * Grabs the tribe-start-date query param from the url if it exists
+			 * and if its a valid date then adds it to the global window object.
+			 */
+			if ( $start_date ) {
+				$start_date = Dates::build_date_object( $start_date, null, false );
+
+				if ( $start_date ) {
+					$this->data['tribe_start_date'] = $start_date->format( Dates::DBDATEFORMAT );
```

Comment:
Should this be `Dates::DBDATETIMEFORMAT` or will it always be just year, month, day?

Reply:
Looks like that was intentional: https://github.com/the-events-calendar/the-events-calendar/commit/f56e491329dc5d48d87605358c752ae5fab580dd